pkg/codegen/docs/gen.go

```diff
@@ -815,13 +816,17 @@ func (mod *modContext) getProperties(properties []*schema.Property, lang string,
 			propLangName = name
 		}
 
+		propID := strings.ToLower("prop" + lang + propLangName)
```

Comment:
Is the `prop` prefix still needed, though?

Reply:
Don't think this is still needed.